pytentiostat/config_reader.py

```diff
@@ -1,10 +1,23 @@
-from yaml import safe_load
+import yaml
 import datetime
 import time
 import sys
+import os
+
+def parse_config_files(configlocation=""):
+    
+    with open(os.path.join(configlocation,'config.yml'),'r') as stream:
+        data = yaml.safe_load(stream)
+
+    with open(os.path.join(configlocation,'adv_config.yml'),'r') as adv_stream:
+        adv_data = yaml.safe_load(adv_stream)
+        
+    return data, adv_data
 
 def get_output_params():
     
+    data, adv_data = parse_config_files()
```

Comment:
Nice, thanks so much @aplymill7 !!!
Now another advice (so sorry!) but parse config does hard-drive access io which is slow, so it would be actually much better to run it once at the beginning some ``main()`` script so we load the ``data`` and ``adv_data`` objects into memory, then to pass them to these functions, so where we run the main script it would say
```
data, adv_data = parse_config_files()
```
then, for example, the ``get_output_params()`` function would be 
```
def get_output_params(data):
    data_out_name = data['general_parameters']['data_output_filename']
    data_out_path = data['general_parameters']['data_output_path']
    return data_out_name, data_out_path
```
sthg like that.  It also means that we can write test code for it without having to do the file-handling which is a bit of a pain.

Reply:
No problem!

Alright, now updated so that main reads config and passes that data to all functions that need it throughout the code. This way, config is only read from once!